app/views/transcripts/show.json.jbuilder

```diff
@@ -2,7 +2,9 @@ json.extract! @transcript, :id, :uid, :title, :description, :url, :audio_url, :i
 :audio_item_url_title,
 :image_item_url_title
 
-json.transcript_status @transcript.transcript_status, :id, :name, :progress, :description
+unless @transcript.transcript_status.nil?
```

Comment:
@RalucaP What do you think of writing this in a positive form?
```
if @transcript.transcript_status.ni?
```

Reply:
Thanks, @wetoolaguer but I think in this case `unless` it's perfect for the job :)